Sources/Plasma/PubUtilLib/plPipeline/hsG3DDeviceSelector.h

```diff
@@ -240,6 +241,9 @@ class hsG3DDeviceRecord
     bool    IsInvalid() const { return 0 != ( fFlags & kInvalid ); }
 
     std::vector<hsG3DDeviceMode>& GetModes() { return fModes; }
+    
+    hsG3DDeviceMode   GetDefaultMode() const { return fDefaultMode; }
+    void    SetDefaultMode( hsG3DDeviceMode mode ) { fDefaultMode = mode; }
```

Comment:
Where is `GetDefaultMode()` used? I see this being set, but never read

Reply:
@dpogue - there were some changes here. Please review b9ca050cb94836043c295f3f0de74bacde5434cd.